### ISSUE TITLE
Copy string into clipboard when selecting

### ISSUE THREAD

**Issue by Kevinwuu** [jeff-hykin/polacode#2]
After i install your extension, when i select some text in vscode ,it will copy the text into clipboard automatic.

The original Polacode also have this situation, but it just happened when i use polacode.

**Comment by jeff-hykin** [comment]
Yup this is definitely an issue. I'm not sure when I'll be able to fix it though. I'll happily merge a PR if you make one though.
